app/src/main/java/com/github/se/signify/ui/screens/home/ASLRecognition.kt

```diff
@@ -112,13 +88,7 @@ fun ASLRecognition(
                   text = "More on ASL Alphabet",
                   backgroundColor = MaterialTheme.colorScheme.primary)
             }
-      },
```

Comment:
I always wondered why the bottom bar was present on the practice screen, so great job removing it!